laser_scan_matcher/src/laser_scan_matcher.cpp

```diff
@@ -181,6 +183,10 @@ void LaserScanMatcher::initParams()
     use_imu_ = true;
   if (!nh_private_.getParam ("use_odom", use_odom_))
     use_odom_ = true;
+  if (!nh_private_.getParam ("use_tf", use_tf_))
+    use_tf_ = true;
```

Comment:
What I'm seeing down in the usage of this variable (line 839) is that it appears to override the `pred_last_base_offset` if that variable is set by the imu or odom. Should it be possible for these options to be true simultaneously?

Reply:
Right.  I was just following the current pattern, where imu supersedes  odom, which in turn supersedes velocity.

In practice, I don't think you would have multiple of these enabled.